File: Java/Conditional Statements/Exercise/05. Godzilla vs. Kong/GodzillaVsKong.java
import java.util.Scanner;

public class GodzillaVsKong {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        double budget = Double.parseDouble(input.nextLine());
        int extras = Integer.parseInt(input.nextLine());
        double clothesPricePerExtra = Double.parseDouble(input.nextLine());

        double movieDecorPrice = budget * 0.1;

        if(extras > 150){
            clothesPricePerExtra -= clothesPricePerExtra * 0.10;
        }

        double totalExpenses = (extras * clothesPricePerExtra) + movieDecorPrice;

        if(totalExpenses > budget){
            System.out.printf("Not enough money!\nWingard needs %.2f leva more.", totalExpenses - budget);
        }else {
            System.out.printf("Action!\nWingard starts filming with %.2f leva left.", budget - totalExpenses);
        }
    }
}

File: Python/Basics/Conditional Statements/Exercises/GodzillaVSKong.py
decorationCost = 10 #percentage of budget
sale = False
saleAmount = 0
costs = 0

budget = float(input())
extras = int(input())
outfitCostPerExtra = float(input())

if extras > 150:
    sale = True
    saleAmount = 10

costs += (budget * (decorationCost / 100))

if sale == True:
    costs += (outfitCostPerExtra * extras) - ((outfitCostPerExtra * extras) * (saleAmount / 100))
else:
    costs += outfitCostPerExtra * extras

if costs <= budget:
    print(f"Action!\nWingard starts filming with {budget - costs:.2f} leva left.")
else:
    print(f"Not enough money!\nWingard needs {costs - budget:.2f} leva more.")